src/containers/about/MeetTheTeam.js

```diff
@@ -0,0 +1,254 @@
+            <TeamMemberRole>{TEAM_ROLES.PIPER}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.JULIA} />
+            <TeamMemberName>{TEAM_NAMES.JULIA}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.JULIA}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.GIULIA} />
+            <TeamMemberName>{TEAM_NAMES.GIULIA}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.GIULIA}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.JASON} />
+            <TeamMemberName>{TEAM_NAMES.JASON}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.JASON}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.BLAKE} />
+            <TeamMemberName>{TEAM_NAMES.BLAKE}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.BLAKE}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.DREW} />
+            <TeamMemberName>{TEAM_NAMES.DREW}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.DREW}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.SHIREEN} />
+            <TeamMemberName>{TEAM_NAMES.SHIREEN}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.SHIREEN}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.ANDY} />
+            <TeamMemberName>{TEAM_NAMES.ANDY}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.ANDY}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.NICHOLAS} />
+            <TeamMemberName>{TEAM_NAMES.NICHOLAS}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.NICHOLAS}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.RYAN} />
+            <TeamMemberName>{TEAM_NAMES.RYAN}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.RYAN}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.TAYLOR} />
+            <TeamMemberName>{TEAM_NAMES.TAYLOR}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.TAYLOR}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.ANDREA} />
+            <TeamMemberName>{TEAM_NAMES.ANDREA}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.ANDREA}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.MANDY} />
+            <TeamMemberName>{TEAM_NAMES.MANDY}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.MANDY}</TeamMemberRole>
+          </TeamMember>
```

Comment:
I think the map () function should be used here.

Reply:
Same deal as: https://github.com/Lattice-Works/openlattice.com/pull/202#discussion_r366556191.